pkg/eventmonitor/eventmonitor.go

```diff
@@ -63,8 +63,13 @@ var _ module.Module = &EventMonitor{}
 
 // EventConsumer defines an event consumer
 type EventConsumer interface {
+	// ID returns the ID of the event consumer
 	ID() string
+	// Start starts the event consumer
 	Start() error
+	// PostStart is called after the event stream is started
+	PostStart() error
```

Comment:
This doesn't seem to speak to me in terms of what the intention of this callback is, maybe a different name?

Reply:
Yeah I'm having some trouble finding a good name for this, would you prefer `AfterEventStreamStart` ?

Reply:
Maybe that, curious what others think. Part of the reason I am struggling is because the use case isn't very clear - the rest of this interface API looks very straightforward (ID, Start, Stop).